packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemWrapperStyles.ts

```diff
@@ -159,6 +159,7 @@ export const menuItemWrapperStyles: ComponentSlotStylesPrepared<MenuItemWrapperS
       },
 
       ...(iconOnly && {
+        borderRadius: v.iconOnlyBorderRadius,
```

Comment:
why not set this directly on HC theme if that's what you're targeting ?

Reply:
The border radius of the focus ring in other themes is affected, is that intentional ? 

![image](https://user-images.githubusercontent.com/20744592/140025234-cb55ae4d-7d41-4045-a94b-4ee24495af3d.png)

Reply:
The focus border was already changed in #20230, so the rounded corners are intentional.

Reply:
> 
> why not set this directly on HC theme if that's what you're targeting ?

Because I think rounded corners would still be correct if you were to add a background in others themes.